<launch>
  <arg name="lio_config_path" default="$(find-pkg-share lidar_inertial_odometry)/config/lidar_inertial_odometry.params.yaml"/>

  <set_parameter name="use_sim_time" value="true"/>
  <node pkg="lidar_inertial_odometry" exec="lidar_inertial_odometry_node" name="lidar_inertial_odometry" output="screen" >
    <remap from="points_raw" to="/sensing/lidar/left_upper/pointcloud_raw"/>
    <remap from="imu_raw" to="/sensing/imu/tamagawa/imu_raw"/>
    <!-- <remap from="points_raw" to="/livox/lidar"/> -->
    <!-- <remap from="imu_raw" to="/livox/imu"/> -->
    <param from="$(var lio_config_path)"/>
  </node>

  <node pkg="rviz2" exec="rviz2" name="rviz2" args="-d $(find-pkg-share lidar_inertial_odometry)/rviz/lidar_inertial_odometry.rviz"/>
</launch>
